src/pages/FlameScans/main.ts

```diff
@@ -99,13 +100,16 @@ export const FlameScans: pageInterface = {
         con.error('404');
```

Comment:
As the page is a single page application now, meaning there is no reload between the pages. $.ready is not enough and It is necessary to listen for page changes and trigger handlePage again accordingly. 

Important navigations:
- Navigation to the overview page
- Navigation to the chapter page
- Chapter changes